staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1826,7 +1826,7 @@ type CSIPersistentVolumeSource struct {
 	// controllerExpandSecretRef is a reference to the secret object containing
 	// sensitive information to pass to the CSI driver to complete the CSI
 	// ControllerExpandVolume call.
-	// This is an beta field and requires enabling ExpandCSIVolumes feature gate.
+	// This is a beta field.
```

Comment:
Same comment about dropping the beta language or keeping a feature gate reference, depending on whether the field is beta or not

Reply:
Same as above. Thanks!